src/style.css

```diff
@@ -0,0 +1,71 @@
+.container {
+  height: 160px;
+  width: 140px;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin-left: 22px;
+  margin-right: 22px;
+}
+
+.stars__star {
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 > :nth-child(n) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--1 > :first-child {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--1 > :nth-child(n+2) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--2 > :nth-child(n-2) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--2 > :nth-child(n+3) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--3 > :nth-child(n-3) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--3 > :nth-child(n+4) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--4 > :nth-child(n-4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--4 > :nth-child(n+5) {
+  background-image: url("./images/star.svg");
+}
+
+.stars--5 > div {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--0 {
+  margin-top: 32px;
+}
+
+.stars--5 {
+  margin-bottom: 32px;
+}
```

Comment:
group these

Reply:
sir I tried that but it didn't work 
because they are not the same styling.